Validations/Validation_2022_w4_ECAL_EGM_PF

```diff
@@ -0,0 +1,66 @@
+# Email/HN link through which this validation is requested.
+# Put is None in case you don't have any link
+ValidationRequest       : None
+
+Title                   : ECAL+EGM+PF updates in view of Run3 start up
+
+Subsystem               : ECAL, EGM, PF
+
+# Labels: List of labels separated by comma. Keep it unique for given validation
+#-------------------------------------------------------------------------------
+Labels                  : Week4, 2022, ECAL, EGM, PF
+
+# Target GTs here
+#-------------------------------------------------------------------------------
+TargetGT_HLT            : 122X_dataRun3_HLTNew_ECAL_EGM_PF_w4_2022_v1
+TargetGT_Express        : 122X_dataRun3_ExpressNew_ECAL_EGM_PF_w4_2022_v1
+TargetGT_Prompt         : 122X_dataRun3_PromptNew_ECAL_EGM_PF_w4_2022_v1
+
+# Reference GTs here
+#-------------------------------------------------------------------------------
+ReferenceGT_HLT         : 122X_dataRun3_HLT_v2
+ReferenceGT_Express     : 122X_dataRun3_Express_v2
+ReferenceGT_Prompt      : 122X_dataRun3_Prompt_v2
+
+# List of Dataset. Comma(,) separated. NO space allowed in between
+#-------------------------------------------------------------------------------
+Dataset                 : /MinimumBias/Commissioning2021-v1/RAW
+
+# Put run-number in JSON format. e.g. {'344518': [[1, 1892]]}
+# Multiple run numbers are NOT supported at the moment
+#-------------------------------------------------------------------------------
+Run                     : {'346512': [[1, 500]]}
+
+# Yes/No. Yes, if you want to submit job for computation if local test is successful
+#-------------------------------------------------------------------------------
+Validate                : Yes
+
+# Choices--> HLT, Prompt, Express, HLT/Prompt, HLT/Express, Prompt/Express, HLT/Prompt/Express
+#-------------------------------------------------------------------------------
+WorkflowsToSubmit       : HLT/Prompt/Express
+
+# Choose the type of condition to submit. 
+# Choices--> New, Reference, Both
+#-------------------------------------------------------------------------------
+NewOrReference          : Both
```

Comment:
Should this be just new and reuse the reference from the validation 2)?

Reply:
Good point, but i'm not sure how to do it ehehhe